docs/database/server/server-v3.sql

```diff
@@ -660,6 +660,9 @@ ON [Post] ([channel_id])
 CREATE INDEX [idx_Post_pending_post_id]
 ON [Post] ([pending_post_id])
 
+CREATE INDEX [idx_Post_type]
+ON [Post] ([type])
+

```

Comment:
I know this is only documentation, but shouldn't it has been updated when you updated the create index?

Reply:
Since this script was generated by [quickdatabasediagrams.com](https://www.quickdatabasediagrams.com), some of the information is not accurate. For example, the index names here (`idx_[table]_[column]`) are not consistent with those from the database (`[table]_[column]`).

So for this change, I simply tried to mimic the syntax of other indices. Although do let me know if you prefer this changed. Thanks!

Reply:
@larkox @theoforger this is interesting.  This file is just a doc, but in reality, we could use this sql file to recreate the entire database.  And the name of the indexes do not match. (idx_Post_type) vs (Post_type).

I think we should keep it consistent so no action is needed on your part @theoforger.

Later on, if we feel the need to match the actual indexes name on db, and on the doc, we will create a PR to update them.